dod-advana/gamechanger-web
# Feature/UOT-109854 JAIC AI Profile Page Development

## Description

- add ts vector to postgres table (with a migration) and use it to search
- (Karolyn) add in review form component
- pull dropdown data from postgres
- populate parts budget profile page (and move some UI around)
- add a new module to the backend (DataHandler, like ExportHandler or SearchHandler)

*there are some added configs for an additional postgres db (called jbook) that will need to be reviewed, and may need to be changed

## Related Issue/Ticket

**MERGE / REVIEW THIS FIRST:** https://github.com/dod-advana/gamechanger-web/pull/191

## Testing instructions

- Navigate to http://10.194.9.105:8080/#/budgetsearch, table should populate on load
- (searching "shark" should yield 1 result)
- click on a result from the results page to be brought to /budgetsearch-profile
- should see some data populating the middle section and table on the right
